src/ngraph/builder/norm.cpp

```diff
@@ -36,125 +37,147 @@ namespace ngraph
+                                                  const AxisSet& reduction_axes)
         {
             // L0 norm returns number of elements different from zero.
-            shared_ptr<Node> zero_node{
-                op::Constant::create(value.get_element_type(),
-                                     value.get_shape(),
-                                     vector<float>(shape_size(value.get_shape()), 0.f))};
+            const shared_ptr<Node> zero_node{ngraph::opset1::Constant::create(
+                value.get_element_type(),
+                value.get_shape(),
+                vector<float>(shape_size(value.get_shape()), 0.f))};
 
             // Convert bool values to input node data type.
-            shared_ptr<Node> non_zero_values = make_shared<op::Convert>(
-                make_shared<op::NotEqual>(value, zero_node), value.get_element_type());
-
-            return make_shared<op::Sum>(non_zero_values, reduction_axes)
+            const shared_ptr<Node> non_zero_values = make_shared<ngraph::opset1::Convert>(
+                make_shared<ngraph::opset1::NotEqual>(value, zero_node), value.get_element_type());
+
+            return make_shared<ngraph::opset1::ReduceSum>(
+                       non_zero_values,
+                       ngraph::opset1::Constant::create(
+                           element::i64, Shape{reduction_axes.size()}, reduction_axes.to_vector()),
+                       false)
                 ->add_provenance_group_members_above({value});
         }
 
-        shared_ptr<Node>
-            l1_norm(const Output<Node>& value, const AxisSet& reduction_axes, float bias)
+        shared_ptr<Node> builder::opset1::l1_norm(const Output<Node>& value,
+                                                  const AxisSet& reduction_axes,
+                                                  float bias)
         {
-            shared_ptr<Node> values{
-                make_shared<op::Sum>(make_shared<op::Abs>(value), reduction_axes)};
+            const shared_ptr<Node> values{make_shared<ngraph::opset1::ReduceSum>(
+                make_shared<ngraph::opset1::Abs>(value),
+                ngraph::opset1::Constant::create(
+                    element::i64, Shape{reduction_axes.size()}, reduction_axes.to_vector()),
+                false)};
 
-            shared_ptr<Node> bias_node{
-                op::Constant::create(values->get_element_type(),
-                                     values->get_shape(),
-                                     vector<float>(shape_size(values->get_shape()), bias))};
+            const shared_ptr<Node> bias_node{ngraph::opset1::Constant::create(
+                values->get_element_type(),
+                values->get_shape(),
+                vector<float>(shape_size(values->get_shape()), bias))};
 
-            return (values + bias_node)->add_provenance_group_members_above({value});
+            return make_shared<ngraph::opset1::Add>(values, bias_node)
```

Comment:
We can relay on NumPy autobroadcasting mechanism for `bias_node` arg.

Reply:
done